config/samples/kafkamanagement_v1beta1_kafkauser.yaml

```diff
@@ -5,6 +5,18 @@ metadata:
 data:
   username: "U2FuY2gtdHdvCg=="
   password: "Qm9oZGFuXyF1YmVyX3VudGVyX3ZhZmVsb2sxNDg4X2lnb3JfdG9saWsxNDg4Cg=="
+#---
+#apiVersion: v1
+#kind: Secret
+#metadata:
+#  name: cert-secret-test
+#data:
+#  clusterId: "YzEzOGQzMmUtZjNlZS00NGViLWE2NjEtODVjOWE0NGQxNTU5Cg=="
+#  commonName: "U2FuY2gtdHdvCg=="
+#  country: "Vk4K"
+#  organization: "SW5zdGFjbHVzdHIK"
+#  organizationalUnit: "SVQK"
+#  validPeriod: "Ngo="
```

Comment:
Why do we need to store this information inside K8s secrets?

Reply:
refactored